README.md

```diff
@@ -236,105 +241,248 @@ Finally, here are the assigned styles:
+You need to add the `ForagePINEditText` component to your app to check a card's 
+balance. Refer to the same instructions for [adding the `ForagePANEditText`](#step-1-add-the-foragepanedittext-ui-component) 
+but change the component name from `ForagePANEditText` -> `ForagePINEditText`
 
-### Step 2: Check the balance of the EBT Card
+### Step 2: Customizing `ForagePINEditText`
+
+For this step, please refer to the same instructions for [customizing the `ForagePANEditText`](#step-2-customizing-foragepanedittext)
+but change the component name from `ForagePANEditText` -> `ForagePINEditText`
+
+### Step 3: Call `setForageConfig()`
+
+In order for ForageElements to work properly, you MUST call `setForageConfig`
+on your ForageElement.
+
+> ⚠️This should be the first method you invoke on a ForagePINEditText instance
+> before calling any other methods
+
+```kotlin
+val tokenizeForagePinEditText = root?.findViewById<ForagePINEditText>(R.id.tokenizeForagePinEditText)
+tokenizeForagePinEditText.setForageConfig(
+    ForageConfig(
+        sessionToken = 'sandbox_AbCsessiontokenXyz',
+        merchantID = '<your_fns_number>'
+    )
+)
+```
+
+#### Parameter Definitions
+- `ForageConfig.merchantID`: A unique seven digit numeric string that [FNS](https://docs.joinforage.app/docs/ebt-online-101#food-and-nutrition-service-fns) issues to authorized EBT merchants.
+- `ForageConfig.sessionToken`: A [session token](https://docs.joinforage.app/reference/create-session-token) that authenticates front-end requests to Forage. To create one, send a server-side request from your backend to the `/session_token/` endpoint.
+
+
+### Step 4: Check the balance of the EBT Card
 
 The ForageSDK exposes the following function to check the balance of an EBT card:
 
 ```kotlin
-    suspend fun checkBalance(
-        context: Context,
-        pinForageEditText: ForagePINEditText,
-        merchantAccount: String,
-        bearerToken: String,
-        paymentMethodRef: String
-    ): ForageApiResponse<String>
+data class CheckBalanceParams(
+    val foragePinEditText: ForagePINEditText,
+    val paymentMethodRef: String
+)
+
+suspend fun checkBalance(params: CheckBalanceParams): ForageApiResponse<String>
 ```
 
-To keep the sensitive information PCI compliant, this function needs the `ForagePINEditText` reference and the `Context` so that Forage can cancel ongoing requests properly. The `paymentMethodRef` parameter is a string identifier that refers to an instance in Forage's database of a [`PaymentMethod`](https://docs.joinforage.app/reference/create-payment-method), a tokenized representation of an EBT Card.
+#### Parameter definitions
 
-This is an example of usage inside an ACC ViewModel:
+- `TokenizeEBTCardParams.foragePiNEditText`: To keep the sensitive information PCI compliant, this function needs the `ForagePINEditText` reference and the `Context` so that Forage can cancel ongoing requests properly.
```

Comment:
need to delete the reference to `Context` as none of `ForageSDK`s submit methods care about this [even in `main` today](https://github.com/teamforage/forage-android-sdk/blob/6af970d657095e80ea5ce07f98b12ba031d6e649/forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt)

Reply:
resolved in [0672ed2](https://github.com/teamforage/forage-android-sdk/pull/105/commits/0672ed2478d1d333710148f91c8161c96c5d7df0)